go/client/getentries.go

```diff
@@ -29,7 +30,7 @@ func (c *LogClient) GetEntries(start, end int64) ([]ct.LogEntry, error) {
 		return nil, errors.New("start should be <= end")
 	}
 	var resp getEntriesResponse
-	err := fetchAndParse(c.httpClient, fmt.Sprintf("%s%s?start=%d&end=%d", c.uri, GetEntriesPath, start, end), &resp)
+	err := fetchAndParse(context.TODO(), c.httpClient, fmt.Sprintf("%s%s?start=%d&end=%d", c.uri, GetEntriesPath, start, end), &resp)
```

Comment:
That's the idea. They're not created, normally, but always handed to you, and there's two "root" contexts, the "background" context, and the "TODO" context. As far as I can tell, the only difference between the two is that the TODO one identifies a site where we'd like to refactor in order to get a context as a parameter.